darwin/torch/dataset.py

```diff
@@ -193,7 +193,7 @@ def get_class_idx(self, index: int) -> int:
         target: torch.Tensor = self.get_target(index)
         return target["category_id"]
 
-    def measure_weights(self, **kwargs) -> np.ndarray:
```

Comment:
Can we run a test, just to make sure that calling `measure_weights` with no `kwargs` is valid?

Reply:
How would I run a test for that? (I don;t know enough to know how to use this :S )